profiles/SamarthSaxena10.json

```
@@ -0,0 +1,6 @@
+{
+  "image":"https://user-images.githubusercontent.com/90990378/211378645-134aba56-6085-4db4-a02b-61d07fc275dd.png",
```

Non-blocking: missing a space here.
```suggestion
  "image": "https://user-images.githubusercontent.com/90990378/211378645-134aba56-6085-4db4-a02b-61d07fc275dd.png",
```